cmd/serve.go

```diff
@@ -100,6 +118,48 @@ func serveCommand() *cli.Command {
 				Sources: cli.EnvVars("SERVER_ADDR"),
 				Value:   ":8105",
 			},
+			&cli.StringFlag{
+				Name:    "mqtt-server",
+				Usage:   "MQTT Server Host and Port",
+				Sources: cli.EnvVars("MQTT_PASSWORD"),
+			},
+			&cli.StringFlag{
+				Name:    "mqtt-client-id",
+				Usage:   "MQTT Client ID",
+				Sources: cli.EnvVars("MQTT_CLIENT_ID"),
+				Value:   randomClientId,
+			},
+			&cli.StringFlag{
+				Name:    "mqtt-user",
+				Usage:   "MQTT Username",
+				Sources: cli.EnvVars("MQTT_USER"),
+			},
+			&cli.StringFlag{
+				Name:    "mqtt-password",
+				Usage:   "MQTT Password",
+				Sources: cli.EnvVars("MQTT_PASSWORD"),
+			},
+			&cli.StringFlag{
+				Name:    "mqtt-topic-prefix",
+				Usage:   "MQTT Topic Prefix. {topic-prefix}/message",
+				Sources: cli.EnvVars("MQTT_TOPIC_PREFIX"),
+				Value:   "signal-api-receiver",
+			},
+			&cli.IntFlag{
+				Name:    "mqtt-qos",
+				Usage:   "MQTT Quality of Service value",
+				Sources: cli.EnvVars("MQTT_QOS"),
+				Value:   1,
+				Validator: func(q int64) error {
+					allowedValues := []int64{1, 2, 3}
+
+					if !slices.Contains(allowedValues, q) {
+						return fmt.Errorf("cannot parse qos %q", q)
```

Comment:
The error is misleading here, consider something like:
```suggestion
						return fmt.Errorf("qos value %d is not allowed, allowed values: %v", q, allowedValues)
```